### PR TITLE
Port more cv tests

### PR DESCRIPTION
**Pull request by oshtaier**
```
pytest tests/foreman/ui_airgun/test_contentview.py -v -k "test_positive_admin_user_actions or test_negative_non_admin_user_actions or test_negative_non_readonly_user_actions"
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.6.5, pytest-4.0.1, py-1.5.3, pluggy-0.8.0 -- /usr/bin/python3
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ashtayer/Desktop/robottelo, inifile:
plugins: services-1.2.1, mock-1.6.3
collecting ... 2019-01-29 16:26:57 - conftest - DEBUG - BZ deselect is disabled in settings

collected 59 items / 56 deselected                                                                                                                                                           

tests/foreman/ui_airgun/test_contentview.py::test_positive_admin_user_actions PASSED                                                                                                   [ 33%]
tests/foreman/ui_airgun/test_contentview.py::test_negative_non_admin_user_actions PASSED                                                                                               [ 66%]
tests/foreman/ui_airgun/test_contentview.py::test_negative_non_readonly_user_actions PASSED                                                                                            [100%]

===================================================================
3 passed, 56 deselected, 2 warnings in 1114.04 seconds ================================================================
```

### CODE REVIEW

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -1263,104 +823,6 @@ def test_positive_delete_version_without_refresh(self):
...
-            # Publish and promote CV to next environment
-            self.content_views.publish(cv_name)
-            self.assertIsNotNone(
-                self.content_views.version_search(cv_name, 'Version 1.0'))
-            status = self.content_views.promote(cv_name, 'Version 1', env_name)
-            self.assertIn('Promoted to {}'.format(env_name), status)
-
-    @run_only_on('sat')
-    @tier2
-    def test_positive_remove_renamed_cv_version_from_default_env(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_contentview.py]: that coverage basically has no value

File: tests/foreman/ui/test_contentview.py [hunk 2]
```diff
@@ -1263,104 +823,6 @@ def test_positive_delete_version_without_refresh(self):
...
-    def test_positive_promote_CV_with_custom_user_role_and_filters(self):
-        """Publish and promote cv with user with custom role and filter
-
-        :id: a07fe3df-8645-4a0c-8c56-3f8314ae4878
-
-        :expectedresults: CV should be published and promoted successfully
-
-        :CaseLevel: Integration
-        """
-        myrole = get_role_by_bz(1306359)
```
Comment (oshtaier) [inline tests/foreman/ui/test_contentview.py]: closed as not a bug

File: tests/foreman/ui/test_contentview.py (hunk 1 above)
Comment (abalakh) [reply tests/foreman/ui/test_contentview.py]: Are you sure? Sounds more like coverage for some specific regression to me.

Comment (oshtaier) [reply tests/foreman/ui/test_contentview.py]: there is no tags that can provide that information. Anyway, they testing lce removal from renamed cv, not even existence, so yeah - that sounds weird

Comment (oshtaier) [reply tests/foreman/ui/test_contentview.py]: As a tester - I just don't feel value from it

File: tests/foreman/ui_airgun/test_contentview.py [hunk 3]
```diff
@@ -2764,3 +2767,297 @@ def test_positive_search_module_streams_in_content_view(session, module_org):
...
+    repo_name = gen_string('alpha')
+    cv_name = gen_string('alpha')
+    cv_new_name = gen_string('alpha')
+    cv_copy_name = gen_string('alpha')
+    user_login = gen_string('alpha')
+    user_password = gen_string('alphanumeric')
+    lce = entities.LifecycleEnvironment(organization=module_org).create()
+    # create a role with all content views permissions
+    role = entities.Role().create()
+    entities.Filter(
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_contentview.py]: please use `create_role_permissions()` from api helpers instead

File: tests/foreman/ui_airgun/test_contentview.py [hunk 4]
```diff
@@ -2764,3 +2767,297 @@ def test_positive_search_module_streams_in_content_view(session, module_org):
...
+        for tab_name in [
+            'details',
+            'versions',
+            'repositories',
+            'filters',
+            'puppet_modules',
+            'docker_repositories',
+            'ostree_content'
+        ]:
+            assert cv[tab_name] is not None
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_contentview.py]: Use `cv.get(tab_name)` instead, as in negative case it won't be `None`, but `KeyError` will be risen.

File: tests/foreman/ui_airgun/test_contentview.py [hunk 5]
```diff
@@ -2764,3 +2767,297 @@ def test_positive_search_module_streams_in_content_view(session, module_org):
...
+    """
+    # create a content view read only user with lifecycle environment
+    # permissions: view_lifecycle_environments and
+    # promote_or_remove_content_views_to_environments
+    user_login = gen_string('alpha')
+    user_password = gen_string('alphanumeric')
+    lce = entities.LifecycleEnvironment(organization=module_org).create()
+    # create a role with content views read only permissions
+    role = entities.Role().create()
+    entities.Filter(
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_contentview.py]: same here

File: tests/foreman/ui_airgun/test_contentview.py [hunk 6]
```diff
@@ -2764,3 +2767,297 @@ def test_positive_search_module_streams_in_content_view(session, module_org):
...
+    user_cv_permissions.remove('view_content_views')
+    user_cv_permissions_entities = [
+        entity
+        for entity in cv_permissions_entities
+        if entity.name in user_cv_permissions
+    ]
+    # ensure I have some content views permissions
+    assert len(user_cv_permissions_entities) > 0
+    assert len(user_cv_permissions) == len(user_cv_permissions_entities)
+    entities.Filter(
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_contentview.py]: and here

File: tests/foreman/ui_airgun/test_contentview.py [hunk 7]
```diff
@@ -2764,3 +2767,297 @@ def test_positive_search_module_streams_in_content_view(session, module_org):
                     module_streams[0]['Name'] == module_stream
                     and module_streams[0]['Stream'] == module_version
             )
+
+
+@tier2
+def test_positive_admin_user_actions(session, module_org, test_name):
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_contentview.py]: mind renaming test name so it won't be so confusing?

**Comment by oshtaier** [comment]
```
pytest tests/foreman/ui_airgun/test_contentview.py -v -k "test_negative_read_only_user_actions or test_positive_non_admin_user_actions or test_negative_non_readonly_user_actions"
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.6.5, pytest-4.0.1, py-1.5.3, pluggy-0.8.0 -- /usr/bin/python3
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ashtayer/Desktop/robottelo, inifile:
plugins: services-1.2.1, mock-1.6.3
collecting ... 2019-02-14 13:40:48 - conftest - DEBUG - BZ deselect is disabled in settings

collected 59 items / 56 deselected                                                                                                                                                           

tests/foreman/ui_airgun/test_contentview.py::test_positive_non_admin_user_actions PASSED                                                                                               [ 33%]
tests/foreman/ui_airgun/test_contentview.py::test_negative_read_only_user_actions PASSED                                                                                               [ 66%]
tests/foreman/ui_airgun/test_contentview.py::test_negative_non_readonly_user_actions PASSED                                                                                            [100%]

=================================================================== 3 passed, 56 deselected, 2 warnings in 620.64 seconds =================================================================
```

**Pull request merged**